lib/ProgramOptions/Parameters.cpp

```diff
@@ -25,20 +25,21 @@
 
 #include <regex>
 
-namespace {
-std::regex const removeComments("#.*$", std::regex::ECMAScript);
-std::regex const removeTabs("^[ \t]+|[ \t]+$", std::regex::ECMAScript);
-}  // namespace
-
-namespace arangodb {
-namespace options {
+namespace arangodb::options {
 
 std::string removeCommentsFromNumber(std::string const& value) {
+  // note: these regex objects are built upon every invocation of this function.
```

Comment:
maybe magic static?
It will be built only once, lazy and thread-safe

Reply:
Done